packages/cloudflare/README.md

```diff
@@ -2,109 +2,8 @@
 
```

Comment:
I think we should keep the content here as it appears on https://www.npmjs.com/package/@opennextjs/cloudflare

What do other think?

Reply:
I don't feel strongly but would also be happy to land this if @vicb is not strongly against it. Avoiding having to maintain two lots of docs is a good idea. That being said, do we have a plan (other than @dario-piotrowicz's suggestion above) to ensure the docs are kept up to date?

Reply:
Sure 🚀